src/pages/Dashboard/Dashboard.Component.jsx

```diff
@@ -148,6 +150,7 @@ export class Dashboard extends Component {
               filterByType={this.changeTypeFilter()}
               changeTime={this.changeTimeFilter()}
               changeMineAll={this.changeAssignedToMeFilter()}
+              roleId={userInfo.roleId}
```

Comment:
I'm not sure there is a need to pass `roleId` as a prop, since `currentUser` can be accessed directly from `IncidentFilter.Component`

Reply:
Thanks, @appcypher. It was difficult to test the different roles when accessing the `currentUser` directly from the `IncidentFilter` component. Some of the existing tests were already failing. Hence, I passed it as a prop, that way I can pass in different roles as props.

Reply:
We should have a sync on this tomorrow and figure it out together.

Reply:
This has been resolved. Thanks @appcypher

Reply:
LGTM